addons/website/tests/test_ui.py

```diff
@@ -174,11 +174,6 @@ def test_snippet_translation(self):
                 'Contact us': 'Contact us in Parseltongue'
             }
         })
-        self.env.ref('web_editor.snippets').update_field_translations('arch_db', {
-            fake_user_lang.code: {
-                'Save': 'Save in fu_GB',
-            }
-        })
```

Comment:
Why this change?

Reply:
I don't know how to change the translation of JS templates in Python. I tried looking for a way, but there is no models for it, it looks like [it is read directly from the disc](https://github.com/odoo/odoo/blob/b7bf4cc2272f8e004e6fa2fa1f7f11c9d8e9ba8e/odoo/tools/translate.py#L1660).

Reply:
I guess I could patch the controller which fetches the translations, but I thought the JS way where I insert the translated term prior to running the snippets menu was a good enough replacement.

Reply:
Mmh indeed, probably ok :+1: Let's ping the test master to see if he has some insight @rdeodoo :open_mouth:

Reply:
Sounds good to me, I also tested functionally and it still works (having the snippets body in the website lang regardless of the user / right panel lang)